course-v2/assets/css/course-v2.scss

```diff
@@ -167,6 +167,14 @@ pre {
   }
 }
 
+.touch-device-only {
+  display: none;
+
+  @media (pointer: coarse) {
+    display: block;
+  }
```

Comment:
I think we will also want to cover devices that have `pointer: none`.
We could use something like:
```
@media (pointer: coarse), (pointer: none) {
    .touch-device-warning {
        display: block;
    }
}
```
or maybe `not` condition with `pointer:fine` --> `display: block`?
We are looking for any devices which is not a desktop or a laptop (or a touch laptop) and thus will likely not be able to run the .html files as expected, so I think it's alright to use this condition.

I did check on a smart TV, and the warning didn't appear there with the current code (because it has `pointer: none`).

Reply:
Thanks! You're right. I'll add that.